docs/interactions/structured-messages/select.md

```diff
@@ -217,5 +217,5 @@ Primary parameters are used by default, however, some parameters are unique or o
 
 | API Property | Type | Description |
 |-|-|-|
-| **`items.title`** | String | The title of the item. *Truncated to 20 characters.* |
-| **`items.payload`** | String | Automatically gets populated as a random hex if blank. |
```

Comment:
Just fix minor discrepancies, `items` should  be preceded by `structured_content` (it's like that on all other structured messages)